frontend/src/pages/Landing/landing.css

```
@@ -0,0 +1,584 @@
+  --dc-mist: rgba(244, 247, 255, 0.72);
+  --dc-electric: #3de0ff;
+  --dc-electric-deep: #1a9fff;
+  --dc-glass: rgba(8, 16, 36, 0.42);
+  --dc-glass-border: rgba(180, 210, 255, 0.22);
+  --dc-glow: 0 0 24px rgba(61, 224, 255, 0.35);
+
+  position: relative;
+  min-height: 100vh;
+  overflow-x: hidden;
+  color: var(--dc-snow);
+  font-family: "Manrope", "Avenir Next", "Segoe UI", sans-serif;
```

Stylelint (17.14.0)

[error] Expected no quotes around "Manrope" (font-family-name-quotes)

(font-family-name-quotes)